packages/utils/src/index.test.ts

```
@@ -1,442 +1,387 @@
+      expect(mockResponse.end).toHaveBeenCalled();
+    });
+  });
+
+  describe("jsonContent", () => {
+    it("should parse JSON content and call the callback", (done) => {
       const req = {
-        headers: { 'content-type': 'application/json' },
-        body: { key: 'value' },
+        headers: { "content-type": "application/json" },
+        body: { key: "value" },
         on: (event: string, callback: any) => {
```

Biome (2.4.9)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)